resources/js/components/ShowReminderModal.vue

```diff
@@ -42,12 +57,23 @@
     export default {
         data() {
             return {
-                reminder: null
+                reminder: null,
```

Comment:
I wonder if it's safer to initialize this as an empty object `{}`.
Definitely something we should investigate.

Reply:
Definitely something we should investigate! I was in doubt whether to initialize with an empty object or not :dizzy_face:  In any case, we should look with other eyes :eyes:

Reply:
Definitely something we should investigate! I was in doubt whether to initialize with an empty object or not :dizzy_face: In any case, we should look with other eyes :eyes: